.travis.yml

```diff
@@ -1,6 +1,15 @@
 os: linux
-dist: trusty
+dist: xenial
 language: python
+sudo: required
+cache:
+  - apt
+env:
+  global:
+    # Adapted from https://github.com/felixduvallet/ros-travis-integration/blob/master/.travis.yml
+    - CI_SOURCE_PATH=$(pwd)
+    - ROS_PARALLEL_JOBS='-j8 -l6'
+    - PYTHONPATH=$PYTHONPATH:$CI_SOURCE_PATH/venv/lib/python3.6/dis-packages
```

Comment:
Given that you added new environment variables, you should look around and make sure the code is consistent. The 3 lines below makes this inconsistent. When I added them, I was not aware of the env global setting, so maybe you should moove all these "export VARIABLE" up here.

Reply:
done